File: LBAppBaseFrameworkOCTest/LBAppBaseFrameworkOCTest/ViewController.m
//
//  ViewController.m
//  LBAppBaseFrameworkOCTest
//
//  Created by ksnowlv on 2024/9/10.
//

#import "ViewController.h"
#import <LBAppBaseFramework/LBAppBaseFramework.h>
#import <LBAppBaseFramework/LBAppBaseFramework-Swift.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    BClass * b = [BClass new];
    [BClass showInfomation:@"aaaa"];
    LBNSTimer* timer =  [LBNSTimer new];
    [timer start];
    
    
}


@end

File: LBAppBaseFrameworkTest/LBAppBaseFrameworkTest/ViewController.swift
//
//  ViewController.swift
//  LBAppBaseFrameworkTest
//
//  Created by ksnowlv on 2024/9/10.
//

import UIKit
import LBAppBaseFramework

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        let timer = LBNSTimer()
        timer.start()
        

        BClass.showInfomation("aaaaa")
        
        let a = AClass()
        a.testAClass("aaa")
    }
}

